Firestore/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
   exist (instead of returning a nil DocumentSnapshot). Code that was previously
   doing `if (snapshot) { ... }` must be changed to
   `if (snapshot.exists) { ... }`.
+- [fixed] Fixed a crash that could happen when the app is shut down in a state
+  where a write batch has been acknowledged by the server but not yet observed.
```

Comment:
"where a write batch has been acknowledged but not yet observed" is a very insider view of the change.

Maybe "... when the app is shut down after a write has been sent to the server but before it has been received on a listener"?

This is slightly a wrong since the user listener callback is immaterial but captures this more in user visible terms. WDYT?

Also add a `(#2237)` on the end?

Reply:
Done.